docs/Documentation/Scripting/Building-Blocks/Objectives-List.md

```diff
@@ -39,14 +39,17 @@ give accurate results. Experiment with this objective a bit to make sure you've
-
-
+| Parameter       | Syntax                                               | Default Value            | Explanation                                                                                                                                                                                                                                                               |
+|-----------------|------------------------------------------------------|--------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| _Block Type_    | [Block Selector](../Data-Formats.md#block-selectors) | :octicons-x-circle-16:   | The block which must be broken / placed.                                                                                                                                                                                                                                  |
+| _Amount_        | Number                                               | :octicons-x-circle-16:   | The amount of blocks to break / place. Less than 0 for breaking and more than 0 for placing blocks.                                                                                                                                                                       |
+| _Safety Check_  | Keyword (`noSafety`)                                 | Safety Check Enabled     | The Safety Check prevents faking the objective. The progress will be reduced when the player does to opposite of what they are supposed to do. Example: Player must break 10 blocks. They place 10 of their stored blocks. Now the total amount of blocks to break is 20. |
+| _Notifications_ | Keyword (`notify`)                                   | Disabled                 | Displays messages to the player each time they progress the objective. Optionally with the notification interval after colon.                                                                                                                                             |
+| _Location_      | loc:location                                         | Optional. Default: none  | Adds an optional location to the objective, only counting blocks broken/placed at the specific location.                                                                                                                                                                  |
+| _Location 2_    | loc2:location                                        | Optional. Default: none  | Adds an optional second location to only count blocks broken/placed in a rectangle between the two specified locations. (Won't have an effect if parameter location isn't set.)                                                                                           |
```

Comment:
I wouldn't call it loc2, but instead try to find a telling name. Thinking along the lines of `cube` or `opposite_location` but I find none of them a really good choice.

Additionally, there is no need to put le last sentence in parentheses, it is important information.